### PR TITLE
Make Optional* work as an input, add tests

### PR DESCRIPTION
**Pull request by mskacelik**
cc @jmartisk

### CODE REVIEW

File: server/implementation/src/main/java/io/smallrye/graphql/execution/datafetcher/helper/AbstractHelper.java [hunk 1]
```diff
@@ -1,16 +1,8 @@
...
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Optional;
-import java.util.Set;
+import java.math.BigInteger;
+import java.util.*;
```
Comment (phillip-kruger) [inline server/implementation/src/main/java/io/smallrye/graphql/execution/datafetcher/helper/AbstractHelper.java:5]: Unnecessary code change. Please do not use * imports.

**Review by phillip-kruger — DISMISSED** [review]
LGTM. I assume your IDE made the * import.

**Review by phillip-kruger — APPROVED** [review]
LGTM ! Thanks

**Comment by jmartisk** [comment]
For context, this fixes the issue in https://github.com/smallrye/smallrye-graphql/issues/703#issuecomment-810629736

**Pull request merged**